doc/whats_new/v0.20.rst

```diff
@@ -212,8 +214,8 @@ Model evaluation and meta-estimators
   :issue:`9304` by :user:`Breno Freitas <brenolf>`.
 
 - Add `return_estimator` parameter in :func:`model_selection.cross_validate` to
-  return estimators fitted on each split. :issue:`9686` by :user:`Aurélien Bellet
-  <bellet>`.
+  return estimators fitted on each split. :issue:`9686` by
```

Comment:
Argh! Please revert all changes unrelated to the present fix!

Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Reply:
Okay, sorry about that, makes sense (FYI, I was trying to make the file use 80 characters per line).